packages/bsync/src/db/migrations/20241205T030533572Z-subs.ts

```diff
@@ -0,0 +1,24 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('subs_op')
+    .addColumn('id', 'bigserial', (col) => col.primaryKey())
+    .addColumn('actorDid', 'varchar', (col) => col.notNull())
+    .addColumn('entitlements', 'jsonb', (col) => col.notNull())
+    .addColumn('createdAt', 'timestamptz', (col) =>
+      col.notNull().defaultTo(sql`CURRENT_TIMESTAMP`),
+    )
+    .execute()
+  await db.schema
+    .createTable('subs_item')
+    .addColumn('actorDid', 'varchar', (col) => col.primaryKey())
+    .addColumn('entitlements', 'jsonb', (col) => col.notNull())
+    .addColumn('fromId', 'bigint', (col) => col.notNull())
+    .execute()
```

Comment:
We could, but it would require a supporting index on the operations table which I'm not sure we want to do because we want to keep that table as lightweight as possible

We could always reconstruct those values through a migration, but we wouldn't be able to quickly query them. So if we need to quickly query them, I'd recommend materializing them

Reply:
Altho these could just be materialized in the dataplane from subscribing to the event stream 🤔 I'm not sure bsync needs them materialized

Reply:
When I built bsync I materialized both mute and notification state just because it was easy to do and might be useful, to avoid backfilling from the beginning of time to bootstrap a new service.  But not critical, so I think it's alright either way.